src/main/java/Exercise1.java

```diff
@@ -0,0 +1,21 @@
+public class Exercise1 {
+
+    public int getLargestCommonDivisor(int integerA, int integerB) {
+        if(integerA == 0 || integerB ==0) {
+            return Math.abs((int) (integerA+integerB));
```

Comment:
Thao tác ép kiểu không cần thiết, sau khi bỏ `(int)`, nhớ bỏ luôn dấu `()`
```suggestion
            return Math.abs(integerA+integerB);
```